src/main/cli/process.ts

```diff
@@ -135,10 +156,23 @@ export function cli(mainFct: () => void) {
                             dialog.showErrorBox(errorTitle, e.toString());
                             process.stderr.write(e.toString() + EOL);
                         }
+                    }).catch(() => {
+                        // ignore
                     });
                 }
-            },
-        )
-        .help()
-        .parse((_PACKAGING === "0") ? process.argv.slice(2) : process.argv.slice(1));
+            } else {
+                app.exit(0);
+            }
+        },
+    )
+    .help();
+
+/**
+ * main entry of thorium
+ * @param main main function to exec
+ * @param processArgv process.argv
+ */
+export function cli(_main: () => void, _processArgv = process.argv) {
+    mainFct = _main;
+    yargs.parse((_PACKAGING === "0") ? _processArgv.slice(2) : _processArgv.slice(1));
```

Comment:
This will be impacted by https://github.com/readium/readium-desktop/pull/607/files